### DISCUSSION TITLE
Error converting to Onnx

### DISCUSSION THREAD

**Discussion by ItCameFr0mMars** [azuritecoin/OnnxDiffusersUI#93] [Q&A] [answered]
Error in question:
```
python convert_stable_diffusion_checkpoint_to_onnx.py --model_path="./aeros" --output_path="model/aeros_onnx"
Traceback (most recent call last):
  File "D:\stable diff\convert_stable_diffusion_checkpoint_to_onnx.py", line 25, in <module>
    from diffusers.onnx_utils import OnnxRuntimeModel
ModuleNotFoundError: No module named 'diffusers.onnx_utils'

```
1 step closer, but I can't get past this error

**Comment by azuritecoin** [comment]
Diffusers updated today and moved `onnx_utils.py` to another folder. You could edit the convert script if you want. Replace `diffusers.onnx_utils` to `diffusers.pipeline.onnx_utils`

**Reply by Daggra1313** [reply]
I actually had this same error. `diffusers.pipeline.onnx_utils` will produce the same error, but changing it to `diffusers` seems to work.

**Reply by azuritecoin** [reply]
Hey so I fixed the bug. Download the latest setup.bat and run `setup.bat -update`

**Comment by azuritecoin** [comment]
I fixed the bug. Download the latest setup.bat and run `setup.bat -update`

**Reply by ItCameFr0mMars** [reply]
@azuritecoin that new script gave a bunch of 404 errors...

**Comment by ItCameFr0mMars** [comment, marked as answer]
I changed `from diffusers.onnx_utils` to `from diffusers`
